linkcovery/cli/__init__.py

```
@@ -21,14 +25,56 @@
+    if background:
+        import subprocess
+        import sys
+        import webbrowser
+
+        log_dir = get_config().get_log_dir()
+        log_file = log_dir / "webui.log"
+        command = [sys.executable, "-m", "uvicorn", "linkcovery.webui.app:app", "--host", host, "--port", str(port)]
+        if reload:
+            command.append("--reload")
+        with open(log_file, "ab") as log_handle:
+            process = subprocess.Popen(command, stdout=log_handle, stderr=log_handle)
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)